Seasons Defense/Assets/Scripts/MenuController.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using TMPro;
+using UnityEngine;
+
+public class MenuController : MonoBehaviour
+{
+    public GameObject sceneTransitionManager;
+    public string playButtonLeadsTo;
+    public TextMeshProUGUI highScoreText;
+
+    public void Start()
+    {
+        highScoreText.SetText("High Score: " + PlayerPrefs.GetInt("HighScore"));
```

Comment:
Let's just add a default here in case the player has not played the game yet.
```suggestion
        highScoreText.SetText("High Score: " + PlayerPrefs.GetInt("HighScore", 0));
```